### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by ahmedtaa**
# In this branch :
- Edit index.html
- Edit style.css
- Edit index.js
- Edit readme.md

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,11 +4,32 @@
...
+                <li>name:100</li>
+                <li>name:50</li>
+                <li>name:20</li>
+            </ul>
+        </section>
+        <section class="form">
+            <h2>Add your score</h2>
+            <form>
+                <input type="text" id="username" required />
+                <input type="number" id="score" required />
```
Comment (elyor-doniyorov) [inline src/index.html]: - [x] I think it would be more comfortable to use for users if we add placeholder text to input fields. What do you think?

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @ahmedtaa, 

Good job so far!

## To Highlight 👏 
- [ ] There no linter errors 💯 
- [ ] Use of webpack to bundle JavaScript. 🥇 
- [ ] Usage of `import` and `export` instead of `require` or `module.exports` ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (ahmedtaa) [reply src/index.html]: Sure I will do it 👌

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi @ahmedtaa ,  

### Your project is complete! 🟢 
Congratulations! 🎉 

Cheers and Happy coding! 👏🏻 👏🏻 

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**